packages/behandlings-unntak/src/panelDefinisjoner/prosessStegPaneler/VedtakProsessStegPanelDef.tsx

```diff
@@ -21,27 +22,28 @@ class PanelDef extends ProsessStegPanelDef {
     aksjonspunktCodes.KONTROLL_AV_MAUNELT_OPPRETTET_REVURDERINGSBEHANDLING,
   ];
 
-  getEndepunkter = () => [
-    unntakBehandlingApi.TILBAKEKREVINGVALG,
-    unntakBehandlingApi.SEND_VARSEL_OM_REVURDERING,
-    unntakBehandlingApi.VEDTAK_VARSEL,
-    unntakBehandlingApi.TILGJENGELIGE_VEDTAKSBREV,
-    // unntakBehandlingApi.DOKUMENTDATA_HENTE,
-  ];
+  getEndepunkter = (featureToggles: FeatureToggles = {}) =>
+    [
+      unntakBehandlingApi.TILBAKEKREVINGVALG,
+      unntakBehandlingApi.SEND_VARSEL_OM_REVURDERING,
+      unntakBehandlingApi.VEDTAK_VARSEL,
+      unntakBehandlingApi.TILGJENGELIGE_VEDTAKSBREV,
+    ].concat(featureToggles.DOKUMENTDATA ? [unntakBehandlingApi.DOKUMENTDATA_HENTE] : []);
```

Comment:
trenger vi feature togglen lenger? @baskevold 
Det er bare for frisinn den ikke skal være aktiv ikke sant?

Reply:
Stemmer det ja. Den kunne ha vært erstattet av en sjekk på frisinn